src/scss/settings/_variables.scss

```diff
@@ -118,9 +118,9 @@ $zIndex: (
 
 // Path vars
 // ==========================================================================
-$img-path                  : '/assets/dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
-$icon-path                 : '#{$img-path}/icons';
-$font-path                 : '/assets/dist/fonts';
+$img-path                  : 'fozzie/dist/img' !default; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
```

Comment:
It's really from `dist/` to `fozzie/dist`, as there was no `assets/` directory in Fozzie. But this is due to the asset sharing change as I mentioned in the top comment here: https://github.com/justeat/gulp-build-fozzie/pull/67

I'm looking for a better way to handle this though; currently when I run `gulp` in Fozzie postcss throws out an error: 
```
[16:08:23] -----------------------------------------------------------------------
[16:08:23] Error in plugin: 'gulp-postcss'
[16:08:23] Asset not found or unreadable: fozzie/dist/img/icons/general/tick.svg
[16:08:23] -----------------------------------------------------------------------
```